fedimint-server/src/consensus/server.rs

```diff
@@ -301,8 +305,7 @@ impl ConsensusServer {
 
             info!(target: LOG_CONSENSUS, "Session {session_index} completed");
 
-            // if the submission channel is closed we are shutting down
-            if self.submission_receiver.is_closed() {
```

Comment:
We should probably retain the shutdown on submission channel closure as well, no?

Reply:
I cannot remember why I put it in there at the time, it’s also only in the single guardian case and not in the consensus part and seems unnecessary to me so I took it out for consistency.